commands/profile.js

```diff
@@ -1,42 +1,80 @@
 import {
+    MessageEmbed,
     CommandInteraction,
     CommandInteractionOptionResolver,
+    User,
 } from 'discord.js';
 import { userTable, minifyRecords } from '../utils/Airtable.js';
 
 const getProfile = async (
     /** @type {CommandInteraction} */ interaction,
     /** @type {CommandInteractionOptionResolver} */ options
 ) => {
-    let targetUsername = options
-        .getString('username')
-        .replace('<@!', '')
-        .replace('>', '');
-    console.log(`Searching for user, ${targetUsername}`);
+    const mentionedUser = options.getMentionable('username', false);
+    /** @type {User} */
+    let targetUser;
+    if (!mentionedUser) {
+        targetUser = interaction.user;
+    } else {
+        targetUser = mentionedUser;
```

Comment:
This should be `mentionedUser.user` for it to work. If you can make that change, I'll merge in :)